File: sdk/samples/WCFSamples/TechnologySamples/Extensibility/Channels/ChunkingChannel/CS/Service/Host.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.ServiceModel;
using System.ServiceModel.Channels;
using Microsoft.Samples.Channels.ChunkingChannel;
namespace TestService
{
    class Host
    {
        public static void Main()
        {
            CustomBinding binding = new CustomBinding();
            ServiceHost host = new ServiceHost(typeof(service), 
                new Uri("net.tcp://localhost:9000/TestService"), 
                new Uri("http://localhost:8000/TestService"));
            host.AddServiceEndpoint(
                typeof(ITestService),
                new TcpChunkingBinding(),
                "ep1");
            host.Open();
            Console.WriteLine("Service started, press enter to exit");
            Console.ReadLine();
            host.Close();
        }
    }
}

File: sdk/samples/WCFSamples/TechnologySamples/Basic/Contract/Service/Stream/VB/service/Host.vb
' Copyright (c) Microsoft Corporation.  All Rights Reserved.

Imports System
Imports System.Collections.Generic
Imports System.Text
Imports System.ServiceModel
Imports System.Configuration

Namespace Microsoft.ServiceModel.Samples

    Class Host

        ' Host the service within this EXE console application.
        Public Shared Sub Main()

            ' Create a ServiceHost for the StreamingService type
            Using serviceHost As New ServiceHost(GetType(StreamingService))

                ' Open the ServiceHostBase to create listeners and start listening for messages.
                serviceHost.Open()

                ' The service can now be accessed.
                Console.WriteLine("The streaming service is ready.")
                Console.WriteLine("Press <ENTER> to terminate service.")
                Console.WriteLine()
                Console.ReadLine()

            End Using

        End Sub

    End Class

End Namespace